ui/admin/app/routes/account/change-password.js

```diff
@@ -11,7 +11,7 @@ export default class AccountChangePasswordRoute extends Route {
 
   @service session;
   @service intl;
-  @service notify;
+
```

Comment:
_Not blocking_
Is there a reason a new line is being left here versus deleting the whole line?